### ISSUE TITLE
Automatically set Minutes field to a default of 00.

### ISSUE THREAD

**Issue by Balaguru4580** [Balaguru4580/pomodorset#6]
Currently, the Hours field and Minutes field both need to be set manually to start timer. 

This is cumbersome when setting timer for 5 hours, 1 hour etc as we need to put 0 in Minutes field, which is redundant. 

So, it would be nice if the Minutes would default to 00 if only the Hours field is set!

**Comment by mehraankush** [comment]
you mean if you set hour to something . minute value automaticly set to be  zero ?

**Comment by Michelle031** [comment]
Can I be assigned this issue?

**Comment by Balaguru4580** [comment]
> you mean if you set hour to something . minute value automaticly set to be  zero ?

Yeah. Unless we manually set the minute value to something, it should just assume zero by default automatically.

**Comment by Balaguru4580** [comment]
@mehraankush let me know if you also want to be assigned to this issue!
